lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -190,9 +214,9 @@ def _get_server_cert(server_cert_getter, self, charm):
             f"{charm}.{server_cert_getter} returned None; continuing with INSECURE connection."
         )
         return
-    elif not isinstance(server_cert, str):
+    elif not (isinstance(server_cert, (str, Path)) and Path(server_cert).is_absolute()):
         raise TypeError(
```

Comment:
At this point, if we use just Path above, this would no longer be a TypeError.

Reply:
Replaced it with ValueError - seems more valid.